src/components/ScenarioParameters/components/ScenarioParametersInputs/GenericSliderInput.js

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import React from 'react';
+import { BasicSliderInput } from '@cosmotech/ui';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+import { TranslationUtils } from '../../../../utils';
+
+const DEFAULT_MIN_VALUE = 0;
+const DEFAULT_MAX_VALUE = 100;
+function getMinValue(parameterData) {
+  if (parameterData.minValue == null) {
+    return DEFAULT_MIN_VALUE;
+  }
+  return parameterData.minValue;
+}
+
+function getMaxValue(parameterData) {
+  if (parameterData.maxValue == null) {
+    return DEFAULT_MAX_VALUE;
+  }
+  return parameterData.maxValue;
+}
```

Comment:
Minor suggestion improvement (even though these lines may be copied from another file :slightly_smiling_face: ), we could rewrite these lines to something like:
```
const getMinValue = (parameterData) => parameterData.minValue ?? DEFAULT_MIN_VALUE;
const getMaxValue = (parameterData) => parameterData.maxValue ?? DEFAULT_MAX_VALUE;
```